src/services/utxoSumForAddress.ts

```diff
@@ -1,25 +1,52 @@
 import {errMsgs, UtilEither} from "../utils";
 import {Pool} from "pg";
+import { UtxoSumResponse } from "../Transactions/types";
 
-export const askUtxoSumForAddresses = async (pool: Pool, addresses: string[]): Promise<UtilEither<string>> => {
+export const askUtxoSumForAddresses = async (pool: Pool, addresses: string[]): Promise<UtilEither<UtxoSumResponse>> => {
   // TODO: support for payment keys
-    const sqlQuery = `
-    SELECT "public"."utxo_view"."value" AS "value"
-    FROM "public"."utxo_view"
-    WHERE "public"."utxo_view"."address" = any(($1)::varchar array)
-    GROUP BY "public"."utxo_view"."value"
-    ORDER BY "public"."utxo_view"."value" ASC
-  `;
+    const adaQuery = `
+      SELECT "public"."utxo_view"."value" AS "value"
```

Comment:
This original query looks strange to me. Shouldn't it sum the value but leave the "group by" and "order by" out? cc @vsubhuman

Reply:
The `group by` and `order by` on the original query indeed have no purpose and they can be safely removed, so I did it

Reply:
Also, should it be `SELECT SUM(...) AS "value"`?

Reply:
Definitely should, totally missed that, sorry! Fixed in the latest commit